R/cloud.R

```diff
@@ -530,15 +530,20 @@ cloud_email <- function(type = c("broken", "failed"), job_id = cloud_job(), pkg
 #' This is automatically set by [cloud_check()] and only lasts for the current R session.
 #' @param job_id If not `NULL`, sets the active `job_id` to the input.
 #' @export
-cloud_job <- function(job_id = NULL) {
-  if (!is.null(job_id)) {
-    cloud_data$job_id <- job_id
+cloud_job <- function(pkg = ".", job_id = NULL) {
```

Comment:
I wasn't sure about argument order. I think this has to gain a `pkg` argument, but as first arg or after `job_id`?

Reply:
I think it should be the second argument, users would hardly ever need to change this. I realize this is a pain with all of the calls, but I don't think it can be helped.

Reply:
If we make this change it seems like you'd hardly ever need to set `job_id`, but I don't know if it's worth changing every function.